hardhat/deploymentInfo.json

```diff
@@ -174,32 +177,35 @@
     "ApeRegistry": {
       "address": "0x36b58F5C1969B7b6591D752ea6F5486D069010AB"
     },
+    "ApeRegistryBeacon": {
+      "address": "0x51A1ceB83B83F1985a81C295d1fF28Afef186E02"
+    },
     "ApeRouter": {
       "address": "0x0355B7B8cb128fA5692729Ab3AAa199C1753f726"
     },
+    "ApeToken": {
+      "address": "0x15135c52D14D98EB746c76F66200e4b0b84851ae"
+    },
     "ApeVaultFactory": {
       "address": "0x8198f5d8F8CfFE8f9C413d98a0A55aEB8ab9FbB7"
     },
+    "ApeVaultFactoryBeacon": {
+      "address": "0x8198f5d8F8CfFE8f9C413d98a0A55aEB8ab9FbB7"
+    },
     "ApeVaultWrapperImplementation": {
       "address": "0xDC11f7E700A4c898AE5CAddB1082cFfa76512aDD"
     },
     "COToken": {
       "address": "0x172076E0166D1F9Cc711C77Adf8488051744980C"
     },
+    "CoSoul": {
+      "address": "0x4EE6eCAD1c2Dae9f525404De8555724e3c35d07B"
+    },
     "FeeRegistry": {
       "address": "0xf4B146FbA71F41E0592668ffbF264F1D186b2Ca8"
     },
     "VaultBeacon": {
       "address": "0x51A1ceB83B83F1985a81C295d1fF28Afef186E02"
-    },
```

Comment:
@bark4mark these are not built as part of our local deployment process, since all the contracts for hedgey are separate. I saw you added these to this repo, but its getting overwritten. What is the process for deploying Hedgey contracts to local chain 1337 if anyone needs to?